ceno_zkvm/src/scheme.rs

```diff
@@ -144,8 +138,10 @@ pub struct ZKVMProof<E: ExtensionField, PCS: PolynomialCommitmentScheme<E>> {
     pub raw_pi: Vec<Vec<E::BaseField>>,
     // the evaluation of raw_pi.
     pub pi_evals: Vec<E>,
-    opcode_proofs: BTreeMap<String, (usize, ZKVMOpcodeProof<E, PCS>)>,
```

Comment:
here we have 
- String: circuit name
- usize: circuit index
to carry information in proof. However a malicious prover can make both inconsistent.